osl/README.md

```diff
@@ -0,0 +1,120 @@
+# MyoProsthesis
+
+## General:
+
+The MyoProsthesis model combineds both the 80-muscle model on Myosuite (https://github.com/MyoHub/myo_sim/tree/main/leg) and OpenSourceLeg (https://neurobionics.robotics.umich.edu/research/wearable-robotics/open-source-leg/), to simulate a trans-feromal amputation model.
+
+## Modifications:
+
+The MyoLeg model was modified as follows:
+
+1. Copied muscle parameters of right leg to left leg for symmetrical muscle parameters
```

Comment:
This change will make things incompatible with myoleg. If we have proper justification for this, this should happen to the base model as well. This needs to have its own PR.

Reply:
The main justification is is that the passive forces on the joints produced by the muscles in a neutral state is different for both legs. In most biomechanics study (unless those that study symmetry specifically), legs are assumed to have symmetrical properties (e.g. length, muscle properties). Controllers produced with a symmetrical assumption will produce varying results in a asymmetric leg model.

I have discussed this issue with Huawei before and he did mention that the optimization process in MyoConverter does not guarantee symmetry, even with the same input parameters, since the parameters of both legs are optimized separately. One thing that was suggested from our discussions is to take the muscle parameters with the least mean passive force.

In this case, should we have a greater discussion with the rest of the myosuite team before deciding to make a PR on the base model?

Reply:
Yes, this needs a larger discussion. For this PR 
[ ] Lets keep the original values 
[ ] Finalize the model on the original values. 
[ ] Get this merged 
For the model update 
[ ] Start a discussion on the appropriate 
[ ] Drive team to concensum 
[ ] Start a new PR for it 
As a general rule of thumb - one change per PR

Reply:
Changed left leg muscle parameters back to original value